sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusClientBuilder.java

```diff
@@ -948,7 +981,31 @@ public ServiceBusReceiverAsyncClient buildAsyncClient() {
          *     queueName()} or {@link #topicName(String) topicName()}, respectively.
          */
         public ServiceBusReceiverClient buildClient() {
-            return new ServiceBusReceiverClient(buildAsyncClient(), retryOptions.getTryTimeout());
+            return new ServiceBusReceiverClient(buildAsyncClient(false), retryOptions.getTryTimeout());
```

Comment:
Same as above. Async client should be built with autocomplete enabled for the processor. We may have to find a different way to disable it on the `receive()` method on the sync client.